Cargo.toml

```diff
@@ -1,47 +1,12 @@
-[package]
-name = "libwaysip"
```

Comment:
Do not change here, this way can keep the version of waysip and libwaysip in the same version, and both can be published to crate